From 72b521be7640a1b06adf6881b12277276ab8609d Mon Sep 17 00:00:00 2001
From: Piotr Tworek <ptworek@vewd.com>
Date: Wed, 06 Apr 2022 19:34:29 +0000
Subject: [PATCH] IWYU: Add vector needed by StatsEntry.

This builds fine with libc++, but fails due to std::vector being
undefined when using libstdc++. Add missing vector include to fix this.

Bug: 957519
Change-Id: I12f91636fe9c69046ae36528e48782acf741b66c
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3557455
Auto-Submit: Piotr Tworek <ptworek@vewd.com>
Reviewed-by: Chrome Cunningham <chcunningham@chromium.org>
Commit-Queue: Chrome Cunningham <chcunningham@chromium.org>
Cr-Commit-Position: refs/heads/main@{#989578}
---

diff --git a/media/capabilities/webrtc_video_stats_db.h b/media/capabilities/webrtc_video_stats_db.h
index 8a5c0cf..0e8653b 100644
--- a/media/capabilities/webrtc_video_stats_db.h
+++ b/media/capabilities/webrtc_video_stats_db.h
@@ -7,6 +7,7 @@
 
 #include <memory>
 #include <string>
+#include <vector>
 
 #include "base/callback_forward.h"
 #include "base/check.h"
